vowpalwabbit/interactions.cc

```diff
@@ -168,6 +210,27 @@ inline bool must_be_left_sorted(const std::vector<namespace_index>& oi)
   return false;  // 'aaa' or 'abc'
 }
 
+std::vector<std::vector<namespace_index>> expand_quadratics_wildcard_interactions(
```

Comment:
this function will always remove true duplicates (ie: `{a, b} {a, b}`). I assume thats desirable?

Reply:
Yes that is desired behavior